CONTRIBUTING.md

```diff
@@ -48,6 +48,34 @@ Use proper judgement, keeping simplicity and user-friendliness as the top priori
 
 When in doubt, have a look at a few existing pages :).
 
+## Directory structure
+
+The English pages directory is `pages`, under which are the platform directories. Language specific directories must follow the pattern `pages.<locale>`, where `<locale>` is a [POSIX Locale Name](https://www.gnu.org/software/gettext/manual/html_node/Locale-Names.html#Locale-Names) in the form of `<language>[_<country>]`, where:
+
+- `<language>` is the shortest [ISO 639](https://en.wikipedia.org/wiki/ISO_639) language code for the chosen language (see [here](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes) for a complete list).
+- `<country>` is the two-letter [ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1) country code for the chosen region (see [here](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements) for a complete list).
+
+[!NOTE]
+> Do not put any page directly inside the English pages directory or language specific directories.
+
+The `<country>` code is optional and should only be added when it is needed. In other words, only when there is a valid reason to distinguish between a language (`ll`) and its regional dialects (`ll_CC1`, `ll_CC2`, etc.). For example, both `fr_FR` and `fr_BE` should fall under the same `pages.fr` directory since there virtually is no difference in writing between standard French and Belgian French.
+
+### Platform directories
+
+The `pages` directory contain the platform directories, as well as the language-specific directories, if they have at least one page of that platform.
+
+The platform directories and their respective pages that need to be put in it are:
+
+- `android`: Android
+- `freebsd`: FreeBSD
+- `openbsd`: OpenBSD
+- `osx`: OSX/Mac OS/macOS (will be replaced by `macos`)
+- `linux`: any Linux distro
+- `netbsd`: NetBSD
+- `sunos`: SunOS
+- `windows`: Windows
+- `common`: available for 2 or more platforms
```

Comment:
```suggestion
- `common`: available for two or more platforms
```

Reply:
This guide is not consistent on in words vs. the figure:
> It is acceptable for several pages to get translated in one pull request.
> While we suggest only two lines for the page description, it is acceptable to have more than two lines if it is necessary to add additional information (i.e. [`pacman`](https://github.com/tldr-pages/tldr/blob/main/pages/linux/pacman.md)).
> In our pages, we use placeholders defined as being tokens within curly brackets. For example, in `sleep {{5}}`, the user can change 5 to any number.
> 1. Try to keep pages at around 5 examples. Pages can be longer or shorter when appropriate but don't exceed the maximum of eight examples.
In what direction should we go?

Reply:
basically all sources recommend to use the "written out" form for number up to ten or even twelve: https://en.wikipedia.org/wiki/English_numerals#Whether_or_not_digits_or_words_are_used

Reply:
@gutjuri in a formal document, yes. Personally I would suggest that the numerical form is more readable.

Reply:
I do not agree, however, I respect the democratic process in this case and will approve this PR as is 😃